docs/universal_transfer_operator/supported_integrations/fivetran/connector/connector.rst

```diff
@@ -0,0 +1,17 @@
+.. _fivetran_connector:
+
+Fivetran Connector
+~~~~~~~~~~~~~~~~~~~
+Fivetran connects to all of your supported data sources and loads the data from them into your :ref:`destination <fivetran_snowflake_destination>`. Each data source has one or more connectors that run as independent processes that persist for the duration of one update. A single Fivetran account, made up of multiple connectors, loads data from multiple data sources into one or more destinations.
```

Comment:
Maybe we can link the supported destination and source here?

Reply:
Added in docs: https://github.com/astronomer/apache-airflow-provider-transfers/pull/62/commits/347ff8d8112284099ebb0fe2debec99b93b016a0